src/screens/Home/Home.js

```diff
@@ -321,6 +322,39 @@ class HomeScreen extends React.Component<Props, State> {
     );
   };
 
+  renderReferral(colors) {
+    const {
+      navigation,
+      smartWalletFeatureEnabled,
+    } = this.props;
+
+    const {
+      isReferralBannerVisible,
+    } = this.state;
+
+    if (!smartWalletFeatureEnabled) {
+      return null;
+    }
+
+    return (
+      <Banner
+        isVisible={isReferralBannerVisible}
+        onPress={() => navigation.navigate(REFER_FRIENDS)}
```

Comment:
As far as I know here we would need to show alert that this is only possible when your email or phone is verified. I'll add it when merging with UI.